applications/slang/slang_gamma_correction/README.md

```diff
@@ -0,0 +1,107 @@
+# Slang Gamma Correction Example
+
+A Holoscan application demonstrating the integration of Slang shading language for GPU-accelerated gamma correction on image data.
+
```

Comment:
Could we please add an image to depict sample output?

Reply:
Added.